### DISCUSSION TITLE
Project is now no longer accepting PRs for Hacktoberfest'22

### DISCUSSION THREAD

**Discussion by pranjay-poddar** [pranjay-poddar/Dev-Geeks#954] [Q&A] [answered]
.

**Comment by pranjay-poddar** [comment, marked as answer]
Sorry guys, due to some spam PRs project is now no longer accepting PRs for Hacktoberfest'22.

We are working on it and hopefully try our best to bring it back for contributions, till then please keep calm and have a great hacktoberfest'22.
Wish you all happy coding✨

**Comment by HuyenSuong20222** [comment]
@ @rachit98  take him in
